test_runner/performance/test_sharded_ingest.py

```diff
@@ -27,14 +34,42 @@ def test_sharded_ingest(
     and fanning out to a large number of shards on dedicated Pageservers. Comparing the base case
     (shard_count=1) to the sharded case indicates the overhead of sharding.
```

Comment:
nit: we don't expect overhead anymore, but rather performance gain, right?

Reply:
We still do pretty much the same amount of CPU work, but send a lot less data over the network and this env is not great at showcasing that.

In any case, actual there's some improvement in total time to ingest and very signifcant improvements in the amount of data marshalled over the network.